libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -379,6 +383,7 @@ dictionary ReverseSwapPairInfo {
     f64 fees_percentage;
     u64 fees_lockup;
     u64 fees_claim;
+    u64? fees_total_estimated;
```

Comment:
NIT: total_estimated_fee

Reply:
It doesn't sound like english to me. I prefer to make it right even if it is not consistent. I don't mind to change other fields as well to be consistent.